# scRNAseq
 
## Resources
[Atlas Mouse/Drodophila/C.elegans](http://atlas.gs.washington.edu/hub/)

[GEO Drop-seq paper](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63473)

##  Tutorials

[Single Cell Informatics Workshop](https://github.com/IMB-Computational-Genomics-Lab/SingleCells2018Workshop)

[scPred](https://joseah.github.io/scPred_docs/)

[SC2P](https://github.com/haowulab/SC2P)

[pagoda2](https://github.com/hms-dbmi/pagoda2/blob/master/vignettes/pagoda2.Rmd)

[dropEst](https://github.com/hms-dbmi/dropEst)

[Single Cell RNA-seq (scRNA-seq) Library Structure](https://teichlab.github.io/scg_lib_structs/)

[Dropviz](http://dropviz.org)

[hemberg-lab](http://hemberg-lab.github.io/scRNA.seq.course/index.html)

[hemberg-lab pdf](http://hemberg-lab.github.io/scRNA.seq.course/scRNA-seq-course.pdf)

## YouTube

[Single Cell Genomics Day 2018 - Overview youtube](https://www.youtube.com/watch?v=X4RBshgH1x0)

[CITEseq youtube](https://www.youtube.com/watch?v=EsyOjkFrr34)

__Techniques__

[Drop-seq](http://mccarrolllab.com/dropseq/)

[Drop-seq papaer](https://www.cell.com/cell/pdf/S0092-8674(15)00549-8.pdf)

[RNA velocity of single cells](https://www.nature.com/articles/s41586-018-0414-6)

[Velocyto](https://github.com/velocyto-team/velocyto.py)

[Scanorama](https://github.com/brianhie/scanorama)

[CITEseq home side](https://cite-seq.com)

[scRNAseq clustering comparison](https://github.com/markrobinsonuzh/scRNAseq_clustering_comparison)

[STARmap](https://github.com/weallen/STARmap)

## 5-prime seq
[Simultaneous multiplexed amplicon sequencing and transcriptome profiling in single cells](https://www.biorxiv.org/content/early/2018/05/22/328328)

### Paper
[scRNA-seq 18,787 individual WTC-CRISPRi](https://genome-cshlp-org.libproxy.temple.edu/content/28/7/1053.full)

[Panoramic stitching of heterogeneous single-cell transcriptomic data](https://www.biorxiv.org/content/early/2018/07/17/371179)

[Estimation of immune cell content in tumour tissue using single-cell RNA-seq data](https://www.nature.com/articles/s41467-017-02289-3)

[Massive single-cell RNA-seq analysis and imputation via deep learning](https://www.biorxiv.org/content/early/2018/07/26/315556)

[A systematic performance evaluation of clustering methods for single-cell RNA-seq data](https://f1000research.com/articles/7-1141/v1)

[Single-cell RNA-seq of human induced pluripotent stem cells reveals cellular heterogeneity and cell state transitions between subpopulations](https://genome.cshlp.org/content/28/7/1053.full)

[Using an atlas of gene regulation across 44 human tissues to inform complex disease- and trait-associated variation](https://www.nature.com/articles/s41588-018-0154-4)

[Integrated Single-Cell Analysis Maps the Continuous Regulatory Landscape of Human Hematopoietic Differentiation](https://www.cell.com/cell/fulltext/S0092-8674(18)30446-X)



--------

# Repeats

## Software

[SalmonTE mapping RE](https://www.rna-seqblog.com/salmonte-an-ultra-fast-and-scalable-quantification-pipeline-for-transposable-elements-from-next-generation-sequencing-data/)

[SalmonTE pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29218879)

[SalmonTE paper](https://psb.stanford.edu/psb-online/proceedings/psb18/jeong.pdf)

[SalmonTE github](https://github.com/hyunhwaj/SalmonTE)

[ERVcaller](http://www.uvm.edu/genomics/software/ERVcaller.html)

[TEcandidates](https://github.com/TEcandidates/TEcandidates)

[RetroSpotter](https://github.com/NCBI-Hackathons/RetroSpotter)

--------
[Dfam](http://www.dfam.org)

[Gypsy Database](http://www.gydb.org)

[Ultraconserved elements (UCEs)](https://www.frontiersin.org/articles/10.3389/fgene.2012.00199/full)

--------

[Grantome: Cell-intrinsic mechanisms of endogenous retroviral control](http://grantome.com/grant/NIH/F30-AI129265-01)

--------

## Papers
[A repetitive elements perspective in Polycomb epigenetics](https://www.frontiersin.org/articles/10.3389/fgene.2012.00199/full)

[Ultraconserved Elements in the Human Genome](http://www.genetics.org/content/192/1/253)

[ERVcaller: Identifying and genotyping non-reference unfixed endogenous retroviruses (ERVs) and other transposable elements (TEs) using next-generation sequencing data](https://www.biorxiv.org/content/early/2018/07/11/332833)

[TEcandidates: prediction of genomic origin of expressed transposable elements using RNA-seq data](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty423/5026658)


# Bioinformatics tutorial

[Bioinformatics DotCa youtube](https://www.youtube.com/channel/UCKbkfKk65PZyRCzUwXOJung)

[Next-Gen Sequence Analysis Workshop (2018)](http://angus.readthedocs.io/en/2018/)

[IHEC tools](http://ihec-epigenomes.org/research/tools)

## Expresion vs Epigenetics

[DEScan2](https://bioconductor.org/packages/release/bioc/vignettes/DEScan2/inst/doc/DEScan2.html)

[Coordinated Gene Activity in Pattern Sets](https://www.bioconductor.org/packages/release/bioc/html/CoGAPS.html)

## Genome Annotation
[Genomation: a toolkit for annotation and visualization of genomic data](https://bioconductor.org/packages/release/bioc/vignettes/genomation/inst/doc/GenomationManual.html)

[Plyranges](https://bioconductor.org/packages/release/bioc/vignettes/plyranges/inst/doc/an-introduction.html)

[Plyranges](https://github.com/sa-lee/plyranges/wiki/API-Design)

[Tximeta](https://github.com/mikelove/tximeta/blob/master/README.md)

[Tximeta slides](https://docs.google.com/presentation/d/1-CdgntDPs5ABc9oiSKlFhDL4hf5Ujz_OyQSNEGQu76Q/edit#slide=id.g3a7280978a_0_19)

[Genomic Ranges Workshop](https://informatics.fas.harvard.edu/genomic-ranges-workshop.html)

[GenomicRanges HOWTOs](http://bioconductor.org/packages/release/bioc/vignettes/GenomicRanges/inst/doc/GenomicRangesHOWTOs.pdf)

[An Introduction to Bioconductor's Packages for Working with Range Data](https://github.com/vsbuffalo/genomicranges-intro/blob/master/notes.Rmd)

[Fluent genomic data analysis with plyranges](https://bioconductor.github.io/BiocWorkshops/fluent-genomic-data-analysis-with-plyranges.html)

[Saying Hello to the Bioconductor Ranges Infrastructure](https://bioconductor.org/packages/release/bioc/vignettes/HelloRanges/inst/doc/tutorial.pdf)

[Segtools](https://segtools.hoffmanlab.org)

## Networks
[netSmooth github](https://github.com/BIMSBbioinfo/netSmooth)

[netSmooth paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5814748/pdf/f1000research-7-16868.pdf)

[netSmooth press](https://www.rna-seqblog.com/netsmooth-network-smoothing-based-imputation-for-single-cell-rna-seq/?utm_campaign=shareaholic&utm_medium=linkedin&utm_source=socialnetwork)

[GeNets](https://www.biorxiv.org/content/early/2018/04/19/196303)

## Pan-Cancer Course

[WEB](https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html?code=cell-site)

[Log in](https://www.workcast.com/ControlUsher.aspx?cpak=7611522012799566&pak=3039550469906299)

[Video Lecture](https://www.workcast.com/AuditoriumAuthenticator.aspx?cpak=7611522012799566&pak=3039550469906299)

## R
[Temple Biostatistics, Fall 2017](http://sjspielman.org/bio5312_fall2017/)

[ggplot2 smootScatter](http://wresch.github.io/2012/11/06/ggplot2-smoothscatter.html)

[The tidyverse: dplyr, ggplot2, and friends](https://monashbioinformaticsplatform.github.io/r-more/topics/tidyverse.html)

[useful R packages](https://support.rstudio.com/hc/en-us/articles/201057987-Quick-list-of-useful-R-packages)

[A brief guide to data visuals in R in 2018](https://www.r-bloggers.com/a-brief-guide-to-data-visuals-in-r-in-2018)

## Stats
[Moving beyond P values: Everyday data analysis with estimation plots](https://www.biorxiv.org/content/early/2018/07/26/377978)

### Varia
[cga-modules](https://github.com/astojmir/cgamodules#adding-new-datasets)

[FANTOM5](http://fantom.gsc.riken.jp/5/)

#### Papers

[Integrative CAGE and DNA Methylation
Profiling](http://mcr.aacrjournals.org/content/molcanres/15/10/1354.full.pdf)

[Transcriptome Analysis of Recurrently Deregulated Genes across Multiple Cancers Identifies New Pan-Cancer Biomarkers](http://cancerres.aacrjournals.org/content/76/2/216.long)

[ISMARA: automated modeling of genomic signals as a democracy of regulatory motifs](https://genome.cshlp.org/content/24/5/869.long)

[ISMARA: automated modeling of genomic signals as a democracy of regulatory motifs](https://www.ncbi.nlm.nih.gov/pubmed/24515121)

-----

# Machine Learning

## Intro
[Understanding Machine Learning](https://medium.com/createdd-notes/understanding-machine-learning-92f13d03d309)

[Machine Learning Notes](https://docs.google.com/document/d/1AISQIb2LVlMzN2tmTbMg3zsoBg9n4xk7HO12hEYgoIw/edit)

[Studying a Course (Machine Learning) with the ADEPT Method](https://betterexplained.com/articles/adept-machine-learning-course/)

[How to build your own Neural Network from scratch in Python](https://towardsdatascience.com/how-to-build-your-own-neural-network-from-scratch-in-python-68998a08e4f6)

[Theano, TensorFlow and the Future of PyMC](https://medium.com/@pymc_devs/theano-tensorflow-and-the-future-of-pymc-6c9987bb19d5)

## Blogs
[Classification from scratch, logistic with splines 2/8](https://www.r-bloggers.com/classification-from-scratch-logistic-with-splines-2-8/)

[Classification from scratch, logistic with kernels 3/8](https://www.r-bloggers.com/classification-from-scratch-logistic-with-kernels-3-8/)


## Sebastian Raschka
[Sebastian Raschka](https://sebastianraschka.com/faq/index.html)

[projects](https://sebastianraschka.com/projects.html)

[Principal Component Analysis in 3 Simple Steps](https://sebastianraschka.com/Articles/2015_pca_in_3_steps.html)

[Machine Learning FAQ](https://sebastianraschka.com/faq/docs/lda-vs-pca.html)

## t-SNE
[T-SNE with PCA](https://www.kaggle.com/jnuic7/t-sne-with-pca)

[Multicore-TSNE](https://github.com/DmitryUlyanov/Multicore-TSNE)

[Results differ from scikit-learn implementation](https://github.com/DmitryUlyanov/Multicore-TSNE/issues/8)

[tSNE python](https://www.linkedin.com/pulse/dimensionality-reduction-using-tsne-python-deepak-kumar/)

[t-SNE Demo](https://blog.applied.ai/visualising-high-dimensional-data/)

[Comprehensive Guide on t-SNE algorithm with implementation in R & Python](https://www.analyticsvidhya.com/blog/2017/01/t-sne-implementation-r-python/)

[How to Use t-SNE Effectively](https://distill.pub/2016/misread-tsne/)

## Dimensionality Reduction
[A Survey of Dimensionality Reduction Techniques
Based on Random Projection](https://arxiv.org/abs/1706.04371)

[A survey of dimensionality reduction techniques](https://arxiv.org/pdf/1403.2877.pdf)

[Dimensionality Reduction and Principal Component Analysis](https://www.commonlounge.com/discussion/fa86fe1108f64c7ab207f4e5bb4e568e)

[Contrastive Principal Component Analysis](https://arxiv.org/pdf/1709.06716.pdf)

[UMAP](https://github.com/lmcinnes/umap)

[Enter the matrix: factorization uncovers knowledge from omics](https://www.biorxiv.org/content/early/2018/04/02/196915)

[Annoy (Approximate Nearest Neighbors)](https://github.com/spotify/annoy#readme)

[How Many Factors to Retain in Factor Analysis](https://www.r-bloggers.com/how-many-factors-to-retain-in-factor-analysis)


#### Lectures
[Mining of Massive Datasets web](http://www.mmds.org/)

[Mining of Massive Datasets Course](https://lagunita.stanford.edu/courses/course-v1:ComputerScience+MMDS+SelfPaced/about)

[Automata Theory](https://lagunita.stanford.edu/courses/course-v1:ComputerScience+Automata+SelfPaced/about)


## Introduction to Data Science
[STOR 390: Introduction to Data Science](https://idc9.github.io/stor390/#reading)

[Data Science Tutorials](https://idc9.github.io/ds_tutorials/)

[Angle based Joint and Individual Variation Explained R](https://github.com/idc9/r_jive)

[Angle based Joint and Individual Variation Explained python](https://github.com/idc9/py_jive)

## BLOGR
[pipelearner](https://drsimonj.svbtle.com/easy-machine-learning-pipelines-with-pipelearner-intro-and-call-for-contributors)

[xgboost and pipelearner](https://drsimonj.svbtle.com/with-our-powers-combined-xgboost-and-pipelearner)

[Visualising Residuals](https://drsimonj.svbtle.com/visualising-residuals)

## Statistical Learning Stanford 
[Statistical Learning Stanford coures](https://lagunita.stanford.edu/courses/HumanitiesSciences/StatLearning/Winter2016/about)

[Introduction to Statistical Learning](https://www.youtube.com/watch?v=St2-97n7atk)

[StatsLearning Lect1/2a](https://www.youtube.com/watch?v=2wLfFB_6SKI&index=2&list=PLxXnVFYbQ9rk0kWidYqrTDgVCw0ae3Yaw)

[An Introduction to Statistical Learning](http://www-bcf.usc.edu/~gareth/ISL/)

[The Elements of Statistical Learning](https://web.stanford.edu/~hastie/ElemStatLearn/)

[Trevor Hastie - Gradient Boosting Machine Learning](https://www.youtube.com/watch?v=wPqtzj5VZus)

## Papers
[Hoffman: Machine Learning for Integrating Data in Biology and Medicine](https://arxiv.org/abs/1807.00123)


--------

# Deconvolution
[TissueEnrich: A tool to calculate tissue-specific gene enrichment](http://bioconductor.org/packages/release/bioc/vignettes/TissueEnrich/inst/doc/TissueEnrich.html)

[Cell-type deconvolution in epigenome-wide association studies: a review and recommendations](https://www.futuremedicine.com/doi/10.2217/epi-2016-0153)

[cell typeâ€“specific gene expression differences in complex tissues](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3699332/)

[Digital sorting of complex tissues for cell type-specific gene expression profiles](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3626856/)

[Loss-function learning for digital tissue deconvolution](https://arxiv.org/pdf/1801.08447.pdf)

[Deconvolution of the white adipose tissue cell types from whole genome gene expression profiles](https://cris.maastrichtuniversity.nl/portal/en/publications/deconvolution-of-the-white-adipose-tissue-cell-types-from-whole-genome-gene-expression-profiles(3250c9e2-1939-4472-ad9a-9c323954b29c)/export.html)

[Multi-tissue constrained spherical deconvolution](http://mrtrix.readthedocs.io/en/3.0_rc1/constrained_spherical_deconvolution/multi_tissue_csd.html)

[Exploiting Gene-Expression Deconvolution to Probe the Genetics of the Immune System](http://journals.plos.org/ploscompbiol/article/figure?id=10.1371/journal.pcbi.1004856.g001)

[RefFreeEWAS: EWAS using Reference-Free DNA Methylation Mixture Deconvolution](https://cran.r-project.org/web/packages/RefFreeEWAS/)

[CellTypeAdjustment](https://github.com/GreenwoodLab/CellTypeAdjustment)

# Epigenetics

## DNA methylation
[DNA methylation loss in late-replicating domains is linked to mitotic cell division](https://www-nature-com.libproxy.temple.edu/articles/s41588-018-0073-4?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+ng%2Frss%2Fcurrent+%28Nature+Genetics+-+Issue%29)

[Late-replicating heterochromatin is characterized by decreased cytosine methylation in the human genome](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205568/)

[methylKit](https://bioconductor.org/packages/release/bioc/vignettes/methylKit/inst/doc/methylKit.html)

[Identification of rare de novo epigenetic variations in congenital disorders](https://www.nature.com/articles/s41467-018-04540-x.pdf)

# Microbiome
[GUT MICROBES JOIN THE FIGHT AGAINST CANCER](https://www.nature.com/magazine-assets/d41586-018-05208-8/d41586-018-05208-8.pdf)

[Tools for extracting individual genomes from metagenomes](https://github.com/KasperSkytte/mmgenome2)

[Microglial control of astrocytes in response to microbial metabolites](https://www.nature.com/articles/s41586-018-0119-x)

## Varia
[Hoffman lab](https://hoffmanlab.org/)

[MetroNome](https://metronome.nygenome.org/metro)

[PANTHER classification system](https://www-nature-com.libproxy.temple.edu/articles/nprot.2013.092)

[CIS-BP](http://cisbp.ccbr.utoronto.ca/TFreport.php?searchTF=T139289_1.02)

[BiGG Models](http://bigg.ucsd.edu/)

### Varia papers
[Direct Promoter Repression by BCL11A Controls the Fetal to Adult Hemoglobin Switch](https://www.cell.com/cell/fulltext/S0092-8674(18)30296-4)

[Somatic mutations precede acute myeloid leukemia years before diagnosis](https://www.nature.com/articles/s41591-018-0081-z)

[Prediction of acute myeloid leukaemia risk in healthy individuals](https://www.nature.com/articles/s41586-018-0317-6)

[Making Mistakes Empowers Cancer Cells](https://www.cell.com/trends/cancer/fulltext/S2405-8033(18)30112-2)

[How does normalization impact RNA-seq disease diagnosis?](https://www.sciencedirect.com/science/article/pii/S1532046418301424)

[Using an atlas of gene regulation across 44 human tissues to inform complex disease- and trait-associated variation](https://www.nature.com/articles/s41588-018-0154-4)

--------
--------

URLs. Roadmap Epigenomics data, 
ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/roadmapepigenomics/; 

BLUEPRINT epigenome project data, ftp://ftp.ebi.ac.uk/pub/databases/blueprint/; 

ENCODE data project, 
http://www.encodeproject.org/; BisSNP easy run procedure, 
http://people.csail.mit.edu/dnaase/bissnp2011/stepByStep.html; 

Picard tools,

http://broadinstitute.github.io/picard. 
Our entire customized workflow ECWorkflows is hosted and freely available at 
https://github.com/uec/ECWorkflows. 

Our annotations including the genomic coordinates of solo-WCGWs and PMD definitions are available at 
https://zwdzwd.github.io/pmd.




